src/main/webapp/index.html

```diff
@@ -108,8 +107,31 @@ <h2>Archlinux user</h2>
                 </p>
             </div>
         </div>
-        <div>Icons made by <a href="https://www.flaticon.com/authors/good-ware" title="Good Ware">Good Ware</a>, 
+        <div id="comments_title"><h1>Comments</h1></div>
+        <div class="comments_page">
+            <div class="comment_section">
+                <table id="com_table">
```

Comment:
nit: Rename to comment_table to be consistent with other names (comment_section and comment_form)